addons/web_editor/static/src/xml/wysiwyg.xml

```diff
@@ -51,6 +51,11 @@
                     }
                 });
             </script>
+            <style type="text/css">
+                html, body {
+                    overflow: unset;
+                }
+            </style>
```

Comment:
I'll let you judge but I think there is just an architecture problem. The base thing is ok: we have a base_frontend.scs file which is added in the *frontend* asset bundle and we then have a *frontend* layout which do use a `#wrapwrap` as mentioned in `base_frontend.scss`. For extra layouts which use base_frontend.scss... I feel there just should be a `#wrapwrap` in there too (like in reports & co I think?). If mass mailing does not want that `#wrapwrap` for some reason, maybe it should not link base_frontend at all :shrug:

But of course, it's fine like that (use the frontend rules but cancel the ones you don't like). Personally, I am also not a fan of the `unset` keyword, I prefer to be explicit with the value I do want, but that's just me :+1: 

Note: however, in stable if you want the users to receive that fix that seems critical, it should be done in an already-linked static scss file or at worst in some JS and then refactored in master.

Reply:
Thanks for taking a look!

There sure is work to be done around mass_mailing's assets but that's clearly out of scope of this fix (and of the editor in general), and quite complicated since we want to pretend the editor is not in fact in an iframe and so we do need all the styles of wherever it's embedded :/ I don't have anything against putting `#wrapwrap` in the iframe though, I don't know what that could potentially change/break, or why it's not in there. :shrug: 

I agree with your remarks. I moved the styles to `mass_mailing.ui.scss` which is part only of `iframe_css_assets_edit`. And I changed the `unset` keyword for `visible` which is the one the browser was setting it to when unset. I'm at a loss why this is the one that works but `auto` will only let us scroll without making the scrollbars visible so...